.github/workflows/generate_chapters.yml

```diff
@@ -18,9 +18,10 @@ on:
     paths:
       - src/content/**
       - src/templates/**
-      - tools/generate/**
+      - src/tools/generate/**
```

Comment:
I am not too familiar with this section of the site. Do we really want to watch here as well? I guess, the answer is going to be "yes", most likely.

Reply:
Yeah I figure if we change that generate scripts, then we should regenerate 😀

If nothing changes then won’t matter and just a bit of a wasted run. But these scripts don’t change often enough for that to be a problem.